MusicNotationKit/MusicNotationKit/MeasureRepeat.swift

```diff
@@ -70,6 +70,16 @@ public struct MeasureRepeat {
     }
 }
 
+extension MeasureRepeat: CustomDebugStringConvertible {
+    public var debugDescription: String {
+        var description = "[ "
+        description += measures.map { $0.debugDescription }.joined(separator: ", ")
```

Comment:
In order to have it be `let`, let's use string interpolation and you can just make a local string property for the measures mapping for readability.